.gitignore

```diff
@@ -87,3 +87,6 @@ target/
 
 # Mypy cache
 .mypy_cache/
+
+
+venv/
```

Comment:
What is this addition for?

Reply:
I added it to ignore the untracked files in my venv/.

Reply:
So do we need it in here?

Reply:
I don't know how it will affect exactly. You want me to change it back?

Reply:
yes, if it doesn't relate to this pull request, take it out.